templates/add-persistence-attribute-SEND_PHONE_MESSAGE/code.js

```diff
@@ -1,25 +1,23 @@
-const { ManagementClient } = require('auth0');
+const { ManagementClient, AuthenticationClient } = require('auth0');
 
 const HTTP_TIMEOUT = 1000; // 1s
 
-// --- AUTH0 ACTIONS TEMPLATE https://github.com/auth0/opensource-marketplace/blob/main/templates/add-persistence-attribute-SEND_PHONE_MESSAGE ---
+// --- AUTH0 ACTIONS TEMPLATE https://github.com/auth0/opensource-marketplace/blob/main/templates/add-persistence-attribute-PASSWORD_RESET_POST_CHALLENGE ---
```

Comment:
I think you want to revert this back to the original comment, likely a copy/paste thing